pkgs/applications/networking/cluster/tfctl/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+buildGoModule rec {
```

buildGoModule supports a pattern called `finalAttrs`, which works in a similar way to `rec`, but enables values to change dynamically when override are made
```suggestion
buildGoModule (finalAttrs: {
```
(dont forget the final `)` at the end!)